src/Compilers/CSharp/Portable/Symbols/Synthesized/SynthesizedLocal.cs

```diff
@@ -24,16 +24,21 @@ internal sealed class SynthesizedLocal : LocalSymbol
 #if DEBUG
         private readonly int _createdAtLineNumber;
         private readonly string _createdAtFilePath;
+#endif
 
         internal SynthesizedLocal(
```

Comment:
>SynthesizedLocal [](start = 17, length = 16)

Consider separating refactoring in this file into a separate PR, I am assuming it is not related to the issue being addressed.